proposals/policies/assertion-id.md

```diff
@@ -0,0 +1,30 @@
+# Web of Things (WoT) Policy (Draft)
+
+## Assertion IDs of WoT Specs
+
+Implementers typically implement multiple WoT specifications and they are encouraged to provide implementation input by asserting whether they implement an assertion or not.
+To do so, they need to associate the assertion id with the assertion text.
```

Comment:
Should it maybe also be “assertion ID” instead of “assertion id” in the text?

Reply:
yes :)

```suggestion
To do so, they need to associate the assertion ID with the assertion text.
```